src/main/java/com/zufar/onlinestore/payment/converter/PaymentConverter.java

```diff
@@ -2,31 +2,12 @@
 
 import com.zufar.onlinestore.payment.dto.PaymentDetailsDto;
 import com.zufar.onlinestore.payment.entity.Payment;
-import org.springframework.stereotype.Component;
+import org.mapstruct.Mapper;
 
-@Component
-public class PaymentConverter {
+@Mapper
+public interface PaymentConverter {
```

Comment:
I would like that all Mapstruct mappers/converter look like this interface. No extra cod, only what we need. Google Mapstruct annotation and other approaches to avoid Mapstruct builders and make our code more clear and clean